quickstart/etcdmigration.md

```diff
@@ -0,0 +1,267 @@
+---
+Title: ETCD Migration Procedure
+---
+
+By following the given steps, you can successfully migrate ETCD from one node to another during maintenance activities like node drain etc., ensuring the continuity and integrity of the ETCD data.
+
+:::note
+Take a snapshot of the ETCD. Click [here](https://etcd.io/docs/v3.3/op-guide/recovery/) for the detailed documentation.
+:::
+
+
+## Step 1: Draining the ETCD Node
+
+1. Assuming we have a three-node cluster with three ETCD replicas and three pools created in the cluster, verify the ETCD pods and pools with the following commands:
```

Comment:
Don't think there is any relation to Mayastor DiskPools here.

Reply:
Yes, we can remove the pools section.